src/ImageProcessor/Utils.py

```diff
@@ -0,0 +1,12 @@
+from osgeo import gdal
+
+def get_corner_coordinates(image_path: str):
```

Comment:
Take a look at my `GeoDataProvider`, might be better suited there and you could probably re-use some of my code, e.g. GeoPoint / GeoRect.